Cargo.toml

```diff
@@ -61,6 +61,13 @@ license-file = "LICENSE"
 keywords = ["bitcoin", "lightning", "chaumian", "e-cash", "federated"]
 
 
+# Workaround: https://github.com/rust-lang/cargo/issues/12457 which causes
```

Comment:
I assume this issue will be addressed by the Rust team at some point, would it be worth adding a TODO comment to remove this eventually?

Reply:
If we forget to remove it, we just won't get debug symbols in build script binaries, which is never going to be a problem.